src/scripts/validation.js

```diff
@@ -51,9 +51,10 @@ const toggleButtonState = (inputList, buttonElement, settings) => {
 
```

Comment:
* Please, delete the `dot` from the class names inside the `settings` object, otherwise the become **selectors** https://snipboard.io/jvXwgU.jpg 

```js
export const settings = {
  formSelector: ".modal__form",
  inputSelector: ".modal__input",
  submitButtonSelector: ".modal__save-button",
  inactiveButtonClass: "modal__save-button_inactive",  // without the dot
  inputErrorClass: "modal__input_error", // without the dot
  errorClass: "modal__error_visible", // without the dot
};

```